### PR TITLE
Update deploy script to update gh-pages

### PR DESCRIPTION
**Pull request by vccortez**
This is an attempt to update the gh-pages branch using the `git stash` command to share the updated `dist/` folder.

**I am not 100% sure that this works, unfortunately I'm unable to test right now, but I hope you guys get the idea.**

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -6,8 +6,17 @@ git config user.name "Travis CI"
 git config user.email "rennierab@gmail.com"
 
 git checkout master
+git stash
+git stash apply
+git add dist/anything.js
+git add dist/anything.min.js
+git commit -m "Update dist [ci skip]"
+
+git checkout gh-pages
```
Comment (vccortez) [inline deploy.sh:15]: I don't know how the environment is setup inside Travis, so I don't know if this is going to work.

**Comment by Rabrennie** [comment]
lets try it

**Comment by Rabrennie** [comment]
doesn't work :(

**Comment by vccortez** [comment]
Did you catch any error?

**Comment by Rabrennie** [comment]
error: pathspec 'gh-pages' did not match any file(s) known to git.

https://travis-ci.org/Rabrennie/anything.js/builds/98707762 not sure if you can see this

**Pull request merged**